src/CreatorTokenSwapRouter.sol

```diff
@@ -5,6 +5,9 @@ import {IUniversalRouter} from "src/lib/IUniversalRouter.sol";
 import {ICreatorToken} from "src/interfaces/ICreatorToken.sol";
 import {IERC20} from "openzeppelin/token/ERC20/IERC20.sol";
 
+/// @title CreatorTokenSwapRouter
+/// @dev A contract for swapping ETH to USDC and then buying Creator Tokens.
+/// @notice Make sure to get a quote for ETH to Creator Token conversion before interacting.
```

Comment:
It looks like the content for `@dev` and `@notice` may be swapped here?

Reply:
yup! fixed in [e159a53](https://github.com/showtime-xyz/creator-tokens/pull/46/commits/e159a53031291a7ef77626ca2d681aec08053372)